lib/coffeescript/coffeescript.js

```diff
@@ -99,7 +99,7 @@
     }
     sources[filename].push(code);
     if (generateSourceMap) {
-      map = new SourceMap;
+      map = new SourceMap();
```

Comment:
Since `new` always ends up being a `Call` now, we're generating eg `new SourceMap()` from `new SourceMap`, which is in line with eg Prettier's formatting of simple no-arg `new`